digitalocean_plugin/tasks.py

```diff
@@ -59,56 +64,76 @@ def first_unless_none(param, load_func):
 
```

Comment:
In comment above: https://github.com/cloudify-cosmo/cloudify-digitalocean-plugin/pull/2/files#r27453810

You can use a generic def to return names.